scylla/src/transport/load_balancing/default.rs

```diff
@@ -1845,12 +1891,14 @@ mod latency_awareness {
                 retry_period,
                 update_rate,
                 minimum_measurements,
+                scale,
             } = self;
             LatencyAwareness::new_for_test(
                 exclusion_threshold,
                 retry_period,
                 update_rate,
                 minimum_measurements,
+                scale,
```

Comment:
Would it make sense to add some tests for the `scale` parameter?

Reply:
It would probably involve some meticulous calculations to find the penalisation threshold for given parameters (involving scale) and then testing that a node is penalised iff the threshold is crossed. I don't consider it worth our effort.